starter-repos/app-template/config/base/kustomization.yaml

```diff
@@ -30,11 +29,20 @@ vars:
   fieldref:
     fieldpath: metadata.namespace
 
+- name: REDIS_INSTALL_NAMESPACE
```

Comment:
The var `NAMESPACE` and `REDIS_INSTALL_NAMESPACE` are the same, just need to keep one of them.

Reply:
done.

Reply:
I think you can remove `REDIS_INSTALL_NAMESPACE` now.